bundle/src/main/java/com/adobe/acs/commons/http/injectors/AbstractHtmlRequestInjector.java

```diff
@@ -35,10 +35,10 @@
 
 import org.apache.commons.lang.StringUtils;
 import org.apache.commons.lang3.ArrayUtils;
-import org.apache.felix.scr.annotations.Deactivate;
 import org.apache.sling.api.SlingHttpServletRequest;
 import org.osgi.framework.ServiceRegistration;
 import org.osgi.service.component.ComponentContext;
+import org.osgi.service.component.annotations.Deactivate;
```

Comment:
Because this is a public abstract class, I recommend leaving the SCR annotation in place here, in addition to the DS annotation, so that consumers using Felix SCR tooling will still find the old annotation in the class file.